apis/Google.Cloud.Logging.Console/README.md

```diff
@@ -28,3 +28,12 @@ The server supports four URLs:
 - `/async-exception` throws an exception from an async method (which
   calls other async methods)
 
+## Log trace correlation
+
+To see log trace correlation in action in the SampleApp deployed in Google Cloud Run,
+ensure that the `TraceGoogleCloudProjectId` is set to a valid Google Cloud Project ID
+where trace data is being exported to. This can be done by setting the `TraceGoogleCloudProjectId`
+field in [appsettings.json](SampleApp/appsettings.json) before deployment.
```

Comment:
In Cloud Run this is very likely to be the same project where Cloud Run is running. You can check if there's a way to configure Cloud Run to store its own traces on a different project.

But also, this does not need to be deployed in Cloud Run, just in an environment that's configured to export traces, of which Cloud Run is an example.

Reply:
Ah OK Cloud Run is fine then if this was meant for it anyways. Same applies here, let's get the actual docs clean and clear, and them we'll word this one.